src/client/Microsoft.Identity.Client/AppConfig/AbstractApplicationBuilder.cs

```diff
@@ -190,6 +190,17 @@ internal T WithUserTokenCacheInternalForTest(ITokenCacheInternal tokenCacheInter
             return (T)this;
         }
 
+        /// <summary>
+        /// Enables or disables ADAL cache.
+        /// </summary>
+        /// <param name="disableAdalCache">Flag that indicates whether the ADAL cache operations will be invoked.</param>
+        /// <returns></returns>
+        public T WithDisabledAdalCache(bool disableAdalCache = true)
```

Comment:
Most of our `With` methods are positive, i.e. enableX, so I suggest `WithEnableAdalCacheCompatibility(bool enable=true)`. The default has to be `enable` because of backwards compat.

Reply:
Updated.